yorkie/src/main/kotlin/dev/yorkie/document/time/TimeTicket.kt

```diff
@@ -15,6 +15,8 @@ internal data class TimeTicket(
             ?: delimiter.compareTo(delimiter)
     }
 
+    fun toIDString() = "$lamport:$actorID$delimiter"
```

Comment:
I don't think we need this since we are using TimeTicket instance for keys.

Reply:
fixed it in currently opened PR [76352db](https://github.com/yorkie-team/yorkie-android-sdk/pull/12/commits/76352dbb30205f4f0dc187d88ef86071734b83ae)